go-controller/pkg/cni/cni.go

```diff
@@ -142,14 +144,23 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, clientset *ClientSet, useOVS
 	}
 	// Get the IP address and MAC address of the pod
 	// for DPU, ensure connection-details is present
+<<<<<<< HEAD
+	pod, annotations, podNADAnnotation, err := getPodWithAnnotations(pr.ctx, podLister, kclient, namespace, podName,
+=======
 	podUID, annotations, podNADAnnotation, err := GetPodAnnotations(pr.ctx, clientset, namespace, podName,
+>>>>>>> ovn-org/master
```

Comment:
!!

Reply:
@flavio-fernandes missed this conflict when merging master in, took care of it in a last commit. Currently cleaning up commit history